docs/it/week08/08-1.md

```diff
@@ -0,0 +1,192 @@
+---
+lang-ref: ch.08-1
+lang: it
+title: Metodi contrastivi nei modelli basati sull'energia
+authors: Vishwaesh Rajiv, Wenjun Qu, Xulai Jiang, Shuya Zhao
+date: 23 Mar 2020
+translation-date: 10 May 2020
+---
+<!-- title: Contrastive Methods in Energy-Based Models -->
+
+Il dr. LeCun ha passato i primi 15 min circa dando un ripasso dei modelli basati sull'energia. Si faccia riferimento alle lezioni della settimana scorsa (Settimana 7) per queste informazioni, specialmente per i concetti riguardanti i metodi di apprendimento contrastivo.
```

Comment:
Magari senza "Il"?

Reply:
In italiano scrivere Dr. LeCun senza l'articolo determinativo prima non so quanto corretto sia, suona molto da traduzione di Google Translate.
Ti linko due articoli recenti a supporto:
https://www.lastampa.it/la-zampa/cani/2020/04/15/news/lo-studio-canadese-sui-cani-randagi-serbatoio-di-coronavirus-un-ipotesi-fatta-al-computer-senza-prove-1.38721209
"la domanda e risposta che si è fatto il Dr. William Schaffner"

https://www.corriere.it/esteri/20_maggio_26/coronavirus-usa-100-mila-vittime-ora-tocca-campagne-senza-lockdown-455024ee-9f8a-11ea-bcda-1b088225c4d4.shtml
"Il 30 marzo scorso il dottor Anthony Fauci"

Reply:
😵 Credo di starmi dimenticando la mia lingua madre…